cmd/version-enable.go

```diff
@@ -47,6 +58,13 @@ FLAGS:
 EXAMPLES:
   1. Enable versioning on bucket "mybucket" for alias "myminio".
      {{.Prompt}} {{.HelpName}} myminio/mybucket
+
+  2. Enable versioning on bucket "mybucket" while excluding versioning on a few select prefixes.
+     {{.Prompt}} {{.HelpName}} myminio/mybucket --excluded-prefixes "app1/*/_temporary/,app2/*/_staging/"
```

Comment:
String array is easier for the users I guess:
`--exclude-prefix "app1/*/_temporary/" --exclude-prefix "app2/*/_staging/"`

Reply:
a comma-separated is fine for now I think